generators/server/templates/src/main/resources/config/_application.yml

```diff
@@ -108,6 +108,11 @@ security:
     basic:
         enabled: false
 
+server:
+    session:
```

Comment:
It's not needed, but generally it's good to keep cookies httpOnly so that JS code can not read value.